docs/cn/Running-Tachyon-Fault-Tolerant-on-EC2.md

```diff
@@ -0,0 +1,146 @@
+---
+layout: global
+title: 在EC2上运行带容错机制的Tachyon
+nickname: 运行带容错机制的Tachyon
+group: User Guide
+priority: 4
+---
+
+使用Tachyon自带的[Vagrant脚本](https://github.com/amplab/tachyon/tree/master/deploy/vagrant)可以将带容错机制的Tachyon部署在Amazon EC2上。该脚本允许你创建，配置以及销毁集群，该集群自动配置了HDFS相关项。
+
+# 前期准备
```

Comment:
It seems this section is used in multiple docs. What do you think of extracting it into a snippet under `_includes`?